index.html

```diff
@@ -14,27 +14,172 @@
       }
       gtag("js", new Date());
       gtag("config", "G-YJ2WSX508J");
-    </script>
+    </script> -->
 
```

Comment:
- Kindly remove the unused commented lines of code to make your codebase look cleaner.